.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - develop
       - main
+      - beta-braavos-mobile
```

Comment:
Do we need this here?

Reply:
will remove with the next release @Cussone